frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/cover_title.dart

```diff
@@ -220,6 +221,13 @@ class _InnerCoverTitleState extends State<_InnerCoverTitle> {
               .read<ViewBloc>()
               .add(ViewEvent.rename(titleTextController.text));
         }
+        try {
```

Comment:
In which cases will the read function throw an error? Can you use `?` instead of `try-catch` if `ViewInfoBlock` is null in certain cases?